apps/web/app/(app)/environments/[environmentId]/integrations/airtable/components/AddIntegrationModal.tsx

```diff
@@ -291,7 +291,7 @@ export const AddIntegrationModal = ({
               <div className="space-y-4">
                 <div>
                   <Label htmlFor="Surveys">Questions</Label>
-                  <div className="mt-1 rounded-lg border border-slate-200">
+                  <div className="mt-1 max-h-[350px] overflow-y-scroll rounded-lg border border-slate-200">
```

Comment:
350px seems too much, as it still breaks on my machine (with a high resolution), can we adjust it based on screen height and use vh accordingly ?😊🤔

<img width="919" alt="Screenshot 2024-06-26 at 10 09 45 AM" src="https://github.com/formbricks/formbricks/assets/67850763/20da5e5a-1573-46cf-8421-7cf36820fea6">